src/panels/lovelace/editor/config-elements/hui-map-card-editor.ts

```diff
@@ -15,15 +15,17 @@ import {
 import memoizeOne from "memoize-one";
 import { fireEvent } from "../../../../common/dom/fire_event";
 import { hasLocation } from "../../../../common/entity/has_location";
+import { computeDomain } from "../../../../common/entity/compute_domain";
 import "../../../../components/ha-form/ha-form";
 import { SchemaUnion } from "../../../../components/ha-form/types";
+import type { SelectSelector } from "../../../../data/selector";
 import "../../../../components/ha-formfield";
 import "../../../../components/ha-switch";
+import "../../../../components/ha-selector/ha-selector-select";
 import { HomeAssistant, ValueChangedEvent } from "../../../../types";
 import { DEFAULT_HOURS_TO_SHOW, DEFAULT_ZOOM } from "../../cards/hui-map-card";
 import { MapCardConfig } from "../../cards/types";
 import "../../components/hui-entity-editor";
-import "../../components/hui-input-list-editor";
```

Comment:
I believe this component (hui-input-list-editor) is now completely unused.

Reply:
Indeed. Some custom cards may still be using it though. Maybe add a deprecation warning to it?

Reply:
I would remove it completely and then custom cards can update during beta. Internal components should not be used by custom cards as we can change them at any time.

Reply:
Ok I pushed a delete. Feels a little weird to me to trash it (might be useful someday?) but hopefully someone remembers how to find this if it's ever wanted again.